src/browser-frontend/views/browser/chrome/tabbar/tab/tab-close-button.jsx

```diff
@@ -31,7 +31,8 @@ export default class TabCloseButton extends PureComponent {
   handleClick = (e) => {
     this.props.dispatch(PagesModelActions.tabbar.closeTabAnimated({
       pageId: this.props.pageId,
-      removePageAfterMs: 300,
+      removePageAfterMs: 200,
```

Comment:
Is this enough time for all the animations to finish? I see transitions of 0.5s, wouldn't it make more sense to have this delay longer or equal to the transition duration? If it's less, the tab will disappear before the animation finishes.